release-notes.txt

```diff
@@ -0,0 +1,237 @@
+BISDN Linux
+v4.5.0 Release Notes
+Date: 2022-03-15
+
+! New in this release
+- baseboxd:
+  - updated version to 1.11.1
+  - added support for Spanning Tree Protocol (STP, RSTP and MSTP) on bonded interfaces.
+  - added initial support for Broadcom KNET interfaces.
+- Linux kernel updated to version 5.15.22
+- iproute2: udpate to version 5.16
+
+! Experimental features
+- OF-DPA: initial support for creating and enabling Broadcom KNET interfaces.
+
+! Bug fixes
+- fixed bug related to terminaton MAC entries not being correctly removed on IP deletion. (https://github.com/bisdn/basebox/issues/362)
+- resolved STP issue where the initial STP state was missed, causing ports to forward/flood instead of blocking. (https://github.com/bisdn/bisdn.github.io/pull/136)
```

Comment:
i think the link does not fit here

Reply:
Updated to correct pull request in basebox